src/settings-ui/Settings.UI/SettingsXAML/Views/AdvancedPaste.xaml

```diff
@@ -69,18 +73,22 @@
                                 </StackPanel>
                             </tkcontrols:SettingsCard.Description>
                         </tkcontrols:SettingsCard>
+                        <InfoBar
+                            x:Uid="GPO_SettingIsManaged"
```

Comment:
Maybe move befor EnablePasteWithAi setting and change text to: "Paste with AI is managed ..."